### DISCUSSION TITLE
What are the differences between `disable` and `is_supported`?

### DISCUSSION THREAD

**Discussion by nyngwang** [nvim-treesitter/nvim-treesitter#4249] [Q&A] [answered]
I know that `disable` can be passed with `(lang, bufnr)`. Is this the only difference?

**Comment by theHamsta** [comment, marked as answer]
`Disable` is your/the users choice and `is_suported` analyzes the installation an tells whether a feature would be possible. You can provide your own disable function but that would not make sense for is_supported.
